charts/nutanix-csi-storage/values.yaml

```diff
@@ -58,6 +58,18 @@ dynamicFileClassRetention: "Delete"
 # value are: node, volume, file, dynfile
 defaultStorageClass: none
 
+# VolumeSnapshotClass settings
+#
+# Allow creation of volumesnapshotclass, If set to true will create a volumesnapshotclass
+# Decide deletion policy
+createVolumeSnapshotClass: true
+volumeSnapshotClassName: "nutanix-snapshot-class"
+volumeSnapshotClassRetention: "Retain"
+
+# Snapshotter container deployment
+# Decide if snapshooter container should be deployed as a sidecar container along with controller.
```

Comment:
```suggestion
# set it to true, if external-snapshotter sidecar should be deployed along with controller. 
```

Reply:
It will be better to put this under sidecars.snapshotter
https://github.com/subodh01/helm/blob/b169f4ee92319fc41268ec981e663b276e255ff9/charts/nutanix-csi-storage/values.yaml#L166